ryoanji/src/ryoanji/nbody/upsweep_cpu.hpp

```diff
@@ -81,10 +81,7 @@ void upsweepMultipoles(gsl::span<const cstone::TreeNodeIndex> levelOffset, const
         for (TreeNodeIndex i = start; i < end; ++i)
         {
             cstone::TreeNodeIndex firstChild = childOffsets[i];
-            if (firstChild)
-            {
-                multipole2Multipole(firstChild, firstChild + 8, centers[i], centers, multipoles, multipoles[i]);
-            }
+            if (firstChild) { M2M(firstChild, firstChild + 8, centers[i], centers, multipoles, multipoles[i]); }
```

Comment:
Define 8 as a constant, even if it's called EightSiblings.

Reply:
done